spec/helpers_spec.js

```diff
@@ -0,0 +1,11 @@
+describe('helpers', function() {
+  describe('helperFoo', function() {
+    it('should be defiend as a function', function() {
+      expect(typeof this.helperFoo).toBe('function');
```

Comment:
`toBe` should be fine for simple string equality. It effectively uses `===` to compare.